index.js

```diff
@@ -51,6 +51,17 @@ function wrapTestInZone(testBody) {
     : done => testProxyZone.run(testBody, null, [done]);
 }
 
+const bindDescribe = (originalJestFn) => function () {
+  const eachArguments = arguments;
+  return function (description, specDefinitions) {
```

Comment:
timeout is missing

Reply:
Added `timeout` signature, though it is not used later.

Should we also pass `timeout` to the `originaljestFn.call`?

Reply:
Yup, I think it makes sense :D

Reply:
OK, I refactored a little bit. Now they follow the "arguments mutation => apply arguments" pattern, same as `it` and `test` spy functions.